### ISSUE TITLE
Failed to Authenticate With Peloton Error

### ISSUE THREAD

**Issue by amy26point2** [philosowaffle/peloton-to-garmin#235]
HI All, 
I noticed my most recent workouts from this afternoon were not uploading to Garmin. I realized I was running an out of date version of peloton-to-garmin so I updated. However, the issue seems to be an authentication error with Peloton followed by a lot of CancellationToken responses. When I navigate to the link it throws an Error: 405 Method Not Allowed.

[19:23:00 ERR] Failed to download workouts from Peleoton.
Peloton.Dto.PelotonAuthenticationError: Failed to authenticate with Peloton
 ---> Flurl.Http.FlurlHttpTimeoutException: Call timed out: POST https://api.onepeloton.com/auth/login

**Comment by Vicore1** [comment]
Good evening, can you give us more information about your implementation? Sure as which installation? (windows, docker, etc..)

Also, depending on how old a version you were using the configuration.local.json structure might have changed a little more specifically I would take a look at https://philosowaffle.github.io/peloton-to-garmin/configuration/json.html#peloton-config

Another to to look at is to confirm you peloton account is not locked, if you modified the polling to less then an hour peloton will lock your account pretty quick.

**Comment by amy26point2** [comment]
Thank you for the response. I'm running the Mac python installer. I did notice the new config file, so I was using that, but I did have a shorter polling interval that I copied over from my old config. Maybe that was the problem. I was not locked out of my account on any of my devices or within the app. I did a reboot (which I should have done before posting) and set the polling interval back to the example. I noticed the warnings I saw before in the build process resolved themselves. Everything seems to be working fine now. Thanks again!
